cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MeasureReportTypeIndividualTest.java

```diff
@@ -0,0 +1,122 @@
+package org.opencds.cqf.fhir.cr.measure.r4;
+
+import static org.opencds.cqf.fhir.test.Resources.getResourcePath;
+
+import ca.uhn.fhir.context.FhirContext;
+import java.nio.file.Paths;
+import org.hl7.fhir.r4.model.DateTimeType;
+import org.hl7.fhir.r4.model.MeasureReport.MeasureReportStatus;
+import org.hl7.fhir.r4.model.Period;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.opencds.cqf.fhir.api.Repository;
+import org.opencds.cqf.fhir.cr.measure.r4.Measure.Given;
+import org.opencds.cqf.fhir.cr.measure.r4.utils.TestDataGenerator;
+import org.opencds.cqf.fhir.utility.repository.ig.IgRepository;
+
+/**
+ * This test is to verify and confirm that created Individual report has required format and output
+ *      subject field has reference
+ *      report has date
+ *      report type summary
+ *      has measure url
+ *      has correct status
```

Comment:
I see only 2 tests in this class.  Is the point to capture this list of test cases in a future PR?

Reply:
these tests capture correct type and meta data. scoring type should be checking for correct populations, fixed description